### PR TITLE
JPTT backticks

### PR DESCRIPTION
**Pull request by Hootyoo**
Added a section to JPTT about backticks

### CODE REVIEW

File: dashboard2/settings/jptt.md [hunk 1]
```diff
@@ -164,3 +164,21 @@ You can also comment out elements by adding `#` at the start of a line. In this
...
+let publishedText = “Published on “ + date;
+```
+
+could be expressed as
+
+```
+let publishedText = `Published on ${date}`;
+```
+
+Simple cases like the above (or literals without variables) are supported. The expression will become a tag in the Workbench and in XLIFF exports. JavaScript allows any expression to be added to literals but we recommend you to stick to simple expressions if possible.
```
Comment (zmeggyesi) [inline dashboard2/settings/jptt.md:182]: Does that mean function invocations are not supported at all? If so, that should be mentioned.

**Review by zmeggyesi — CHANGES_REQUESTED** [review]
Can you also add an example JSON Path that matches a template literal, for completeness' sake?

File: dashboard2/settings/jptt.md (hunk 1 above)
Comment (Hootyoo) [reply dashboard2/settings/jptt.md:182]: Well, it did work with

```js
let cica = () => {
    return "cica";
}

let output = `Ez a ${cica()} nagyon aranyos`;

console.log(output);
```

I just don't want to suggest they put something horribly complex in there.

Comment (zmeggyesi) [reply dashboard2/settings/jptt.md:182]: Well, that's good so far. What was the path to get `"cica"` extracted here?

Comment (Hootyoo) [reply dashboard2/settings/jptt.md:182]: It's just `cica` and it works as expected.

**Comment by Hootyoo** [comment]
Can the triple-backtick and coding language syntax be used in these markdowns?

**Comment by zmeggyesi** [comment]
> Can the triple-backtick and coding language syntax be used in these markdowns?

Don't think so, I don't think the Markdown compiler supports syntax highlighting.

File: dashboard2/settings/jptt.md [hunk 2]
```diff
@@ -179,6 +179,28 @@ could be expressed as
 let publishedText = `Published on ${date}`;
 ```
 
-Simple cases like the above (or literals without variables) are supported. The expression will become a tag in the Workbench and in XLIFF exports. JavaScript allows any expression to be added to literals but we recommend you to stick to simple expressions if possible.
+Cases like the above (or literals without variables) are now supported. The expression will become a tag in the Workbench and in XLIFF exports.
```
Comment (zmeggyesi) [inline dashboard2/settings/jptt.md:182]: An `x` tag, right? It'll appear as a single opaque element, and not in a paired `g` tag.

**Review by zmeggyesi — APPROVED** [review]

**Pull request merged**